src/machines/java/com/enderio/machines/common/io/item/MachineInventoryLayout.java

```diff
@@ -293,6 +293,28 @@ public Builder ghostSlot(int count, BiPredicate<Integer, ItemStack> filter) {
                 slot(slot -> slot.guiInsert().filter(filter).stackLimit(currentStackLimit));
             }
             return this;
+
+        }
+
+        /**
+         * Add a preview slot.
+         * This slot cannot be inserted to/extracted from.
+         */
+        public Builder previewSlot() {
+            return previewSlot(1);
+        }
+
+        /**
+         * Add a preview slot.
+         * This slot cannot be inserted to/extracted from.
```

Comment:
fix javadoc to imply multiple slots

Reply:
I don't understand

Reply:
Some of the builder methods have javadoc that says adds a ghost slot (so singular) while having an int parameter for the amount of fields. Change the javadoc of those methods to plural

Reply:
Understood. input and output slots have them. Storage and ghost don't. Should I change them as well ?

Reply:
Yeah, all things that allow multipls slots should be written in plural